CHANGELOG.md

```diff
@@ -5,13 +5,21 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 ## (Unreleased) hipCUB-x.x.x for ROCm 6.4.0
 
 ### Added
+* Added regression tests to `rtest.py`. Regression tests are a subset of tests that caused hardware problems for past emulation environments.
+  * Can be run with `python rtest.py [--emulation|-e|--test|-t]=regression`
 * Added `ForEach`, `ForEachN`, `ForEachCopy`, `ForEachCopyN` and `Bulk` functions to have parity with CUB.
 * Added the `hipcub::CubVector` type for CUB parity.
+* Added `--emulation` option for `rtest.py`
+* Unit tests can be ran with `[--emulation|-e|--test|-t]=<test_name>`
```

Comment:
Which part of this was added? The --emulation option or something else? Or is it the script itself? In which case we can rewrite the first point about regression tests.

Reply:
The --emulation option